test/performance/odp_dma_perf.c

```diff
@@ -150,6 +162,7 @@ static void print_usage(void)
 	       "  -T, --in_seg_type         Input segment data type. Packet by default.\n"
 	       "                            Types:\n"
 	       "                                0: packet\n"
+	       "                                1: buffer\n"
```

Comment:
Perhaps buffer is a bit misleading name. We may actually end up adding support for performing DMA operations with buffers (`odp_buffer_t`) in the future. For example `memory` or `memory address` would be a clearer name and it would better match the API spec. The same comment applies to function naming below.

Reply:
Yes, those are better names, will be fixed in v2.